src/main/java/de/jutzig/github/release/plugin/UploadMojo.java

```diff
@@ -270,7 +278,7 @@ private GHRelease findRelease(GHRepository repository, String releaseNameToFind)
 	 */
 	private static final Pattern REPOSITORY_PATTERN = Pattern.compile(
 			"^(scm:git[:|])?" +								//Maven prefix for git SCM
-			"(https?://github\\.com/|git@github\\.com:)" +	//GitHub prefix for HTTP/HTTPS/SSH/Subversion scheme
+			"(https?://[\\w\\d.-]+/|git@[\\w\\d.-]+:)" +	//GitHub prefix for HTTP/HTTPS/SSH/Subversion scheme
```

Comment:
changed to allow other hostnames. Alphanumeric, dots, dashes. Not perfect but better than nothing.